blazor/datagrid/columns.md

```diff
@@ -91,6 +91,11 @@ The following image represents DataGrid with dynamically build columns,
 
 You can achieve complex data binding in the DataGrid by using the dot(.) operator in the column.field. In the following examples, **Name.FirstName** and **Name.LastName** are complex data.
```

Comment:
In the following examples, **Name.FirstName** and **Name.LastName** are complex data: